components/utils/use-merged-state.ts

```diff
@@ -0,0 +1,38 @@
+import { useRef, useState, useCallback } from 'react'
+//https://raw.githubusercontent.com/react-component/util/master/src/hooks/useMergedState.ts
```

Comment:
use url with commit hash rather than `master` branch `https://github.com/react-component/util/blob/4045d568885d65a5491ab22347682f5e31da31dc/src/hooks/useMergedState.ts`
add space after `//`

Reply:
done